common/txmgr/types/config.go

```diff
@@ -0,0 +1,58 @@
+package types
+
+import "time"
+
+type TxmConfig[UNIT any] interface {
```

Comment:
Is using FEE_UNIT more descriptive than UNIT?

Reply:
I like UNIT to keep it shorter, but I will add a comment to document more specifically what it refers to

Reply:
I like FEE_UNIT better, `UNIT` could mean almost anything